source/_integrations/reolink.markdown

```diff
@@ -204,3 +204,4 @@ However, these cameras can work with this integration through an NVR in which th
 - Older firmware versions do not expose the necessary information the integration needs to function. Ensure the camera is updated to the [latest firmware](https://reolink.com/download-center/) prior to setting up the integration. Note that Reolink auto update and check for update functions in the app/windows/web client often do not show the latest available firmware version. Therefore check the version in the [Reolink download center](https://reolink.com/download-center/) online.
 - Ensure at least one of the HTTP/HTTPS ports is enabled in the [windows](https://reolink.com/software-and-manual/)/web client under `Settings`->`Network`->`Advanced`->`Port Settings`, see [additional instructions](https://support.reolink.com/hc/en-us/articles/900004435763-How-to-Set-up-Reolink-Ports-Settings-via-Reolink-Client-New-Client-) on the Reolink site.
 - On some camera models, the RTMP port needs to be enabled in order for the HTTP(S) port to function properly. Make sure this port is also enabled if you get a `Cannot connect to host` error while one of the HTTP/HTTPS ports is already enabled.
+- As of April 2023 the Reolink cameras do not support sending webhooks to a HTTPS endpoint. HA instances that are only available to your camera over HTTPS (i.e. both the internal and external interface of HA are configured with certificates and use HTTPS) then certain webhooks will not work. For example, the `Visitor` binary sensor from the Video Doorbell will not be updated when the doorbell button is pressed. There is no sanctioned solution to this, see the [github issue](https://github.com/home-assistant/core/issues/91640) for more details.
```

Comment:
- "As of April 2023", did it work before that? I'm confused about why a time is added here.
- We don't track issues in docs, please remove the issue reference.
- Don't abbreviate HA, that is not needed in this sentence.
- "Visitor" is not code, and should not be code fenced.
- 2023.5 (beta start this week), will have detection for this and inform users in the repair center. Not sure if this is needed to be added because of that.

Reply:
> What happens if this doesn't make the beta (or final release)?

That is not a concern of this PR.

> Or if someone if still running an old version

Or documentation `current` branch only reflects the latest stable version. Others are not a consideration or concern.

Reply:
That's my point, I'm trying to document a current issue!

It may or may not be an issue in the next release - but it is _currently_ an issue. :-)